.ci/monolithic-linux.sh

```diff
@@ -49,17 +52,31 @@ trap at-exit EXIT
 
 projects="${1}"
 targets="${2}"
+runtimes="${3}"
 
 lit_args="-v --xunit-xml-output ${BUILD_DIR}/test-results.xml --use-unique-output-file-name --timeout=1200 --time-tests"
 
 echo "--- cmake"
 export PIP_BREAK_SYSTEM_PACKAGES=1
+
 pip install -q -r "${MONOREPO_ROOT}"/mlir/python/requirements.txt
 pip install -q -r "${MONOREPO_ROOT}"/lldb/test/requirements.txt
 pip install -q -r "${MONOREPO_ROOT}"/.ci/requirements.txt
+
+# This is an lldb requirement which is not listed above.
+pip install -q swig
+
+# Set the system llvm-symbolizer as preferred.
+export LLVM_SYMBOLIZER_PATH=`which llvm-symbolizer`
+[[ ! -f "${LLVM_SYMBOLIZER_PATH}" ]] && echo "llvm-symbolizer not found!"
+
+# Set up all runtimes either way. libcxx is a dependency of LLDB.
```

Comment:
What happens here if `libcxx` isn't set in `LLVM_ENABLE_RUNTIMES` and one tries to build/test lldb?

Reply:
The tests which depend on the python bindings will be skipped.
Which are actually the (only?) tests relevant for a clang change.

Reply:
What's the build/test time impact because of this? We need to make sure we account for capacity constraints.

Reply:
I am not sure what impact you mean. This p

Reply:
I am not sure what impact you mean. This patch does not cause any extra tests to be run, lldb testing is still disabled.

There might be small differences in relation to configure time and installing dependencies, but should be few seconds.

Now if lldb gets added back, then a few extra tests are run, but some of that is offset by the fact libcxx will be only built once instead of twice.

Also that the extra lldb tests being run are the ones that are actually useful for clang.

Reply:
I'm not saying that the tests aren't useful. But we presumably have to build the runtimes and then run additional tests. Throughput is pretty important to think about for premerge given we have limited compute, the frequency of commits, and how much testing we are already doing.

Could you collect timing data (preferrably n=3) on testing a lldb change?

Reply:
Here are some numbers for LLDB-only change:

With change:
1) 4m46s - https://buildkite.com/llvm-project/github-pull-requests/builds/168411#_
2) 4m36s - https://buildkite.com/llvm-project/github-pull-requests/builds/168431#01963503-57a4-4934-9de8-f298abe3c432
3) 4m05s - https://buildkite.com/llvm-project/github-pull-requests/builds/168455#01963544-0108-4474-bcfe-781e6facc804

Without change:
1) 3m41s - https://buildkite.com/llvm-project/github-pull-requests/builds/168504#019635ca-f0cc-4ab7-8723-c761a640a701
2) 3m30s - https://buildkite.com/llvm-project/github-pull-requests/builds/168559#01963642-adc3-4e7c-ac07-aaf8e589c53b
3) 3m45s - https://buildkite.com/llvm-project/github-pull-requests/builds/168599#019636c2-d22a-442f-a4a8-d33fd2d95191